internal/charmstore/store_test.go

```diff
@@ -1167,7 +1167,7 @@ func (s *StoreSuite) TestAddCharmWithBundleSeries(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, `charm added with invalid id cs:~charmers/bundle/wordpress-2`)
 }
 
-func (s *StoreSuite) TestAddCharmWithMultipleSeries(c *gc.C) {
+func (s *StoreSuite) TestAddCharmWithMultiSeries(c *gc.C) {
 	store := s.newStore(c, false)
 	defer store.Close()
 	ch := storetesting.Charms.CharmArchive(c.MkDir(), "juju-gui")
```

Comment:
Perhaps a comment saying that the juju-gui charm supports many series?
Or even name the charm such that it's obvious that it supports multiple series?